README.md

```diff
@@ -97,3 +97,33 @@ This script continuously captures JPEGs from a Raspberry PI's CSI camera and wri
 Why not `raspivid` or `raspistill`? Well, at the time of writing `raspivid` doesn't output JPEGs and `raspistill` only works in *stills* mode.
 
 Why Python and not C? Because most of the stuff is done by the GPU, so the insignificant performance gain would not make it worth writing C code. And of course because [picamera](https://picamera.readthedocs.org/) is an amazing library.
+
+---
+
+### HTTP Authentication
+No IP camera should be operated without credentials. Therefore, it is important to secure the camera stream with user name and password.
+In this example, we enable HTTP authentication with these parameters:
+
+    raspimjpeg.py -w 640 -h 480 -r 15 | streameye -a basic -c username:password:workgroup
+    
+If we now call the MJPEG stream URL, we will be asked for username and password
+
+---
+
+### How to start the MJPEG stream automatically with Ubuntu/Raspbian
```

Comment:
I'd leave out this section, given that startup services these days are complex and each user does this their own preferred way, usually via a systemd unit file. Also I think it's beyond the scope of a README for this project.